### DISCUSSION TITLE
Change top level "document types" folder name

### DISCUSSION THREAD

**Discussion by matthewcare** [umbraco/Umbraco-CMS#12838] [Features and ideas]
Element types and compositions have been around for a long time now, but the interface hasn't been updated to follow these new distinctions.

"Document Types" is top level folder item yet you can create items underneath which aren't only new document types.
Perhaps a new name should be given to this folder which better reflects the contents.

This change will allow a better folder structure to split up the different types without nesting a "Document Types" folder within the "Document Types" folder.

Thoughts?

**Comment by leekelleher** [comment]
I'd go with "Content Types".

**Comment by abjerner** [comment]
I'd go for **Content types** as well. **Document types** is a legacy term, which might even be a bit confusing to people new to Umbraco.

I had a chat with Niels Hartvig about the name a few years back, and he agreed when I asked if they shouldn't be called *content types* instead. I've considered creating an issue or PR about it a couple of times after that, but I haven't gotten around to that yet.

If renamed to **Content types**, the name would then also aligns with the content type service in the underlying code.
